modules/@apostrophecms/image/ui/apos/components/AposMediaManagerEditor.vue

```diff
@@ -276,21 +276,23 @@ export default {
   }
 
   .apos-media-editor /deep/ .apos-field {
-    margin-bottom: 20px;
+    margin-bottom: $spacing-double;
   }
 
   .apos-media-editor__details {
     @include apos-list-reset();
+    margin-bottom: $spacing-double;
+  }
+
+  .apos-media-editor__detail {
+    @include typeMeta;
     color: var(--a-base-4);
-    font-size: map-get($font-sizes, default);
-    font-weight: 500;
-    margin-bottom: 20px;
```

Comment:
where'd this margin go if `typeMeta` doesn't include it

Reply:
novella-meta (the only meta after switching to kebob), does not reset the margin